### PR TITLE
Add missing municipality name to suministro details breadcrumb

### PR DESCRIPTION
**Pull request by jonatns**
Fix for #144

### CODE REVIEW

File: suministrospr/suministros/templates/suministros/suministro_detail.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <div class="flex justify-center">
   <div class="flex flex-row text-sm text-gray-600">
     <div class="ml-2">
-      <a href="{% url 'suministro-municipio-list' object.municipality|slugify %}">{{ object.get_municipality_display }}</a>
+      <a href="{% url 'suministro-municipio-list' object.municipality|slugify %}">{{ object.municipality.get_name_display }}</a>
```
Comment (jpadilla) [inline suministrospr/suministros/templates/suministros/suministro_detail.html]: @jonatns Thank you!

Although technically correct, can we change this to:

```suggestion
      <a href="{% url 'suministro-municipio-list' object.municipality.slug %}">{{ object.municipality.name }}</a>
```

**Review by jpadilla — CHANGES_REQUESTED** [review]

File: suministrospr/suministros/templates/suministros/suministro_detail.html (hunk 1 above)
Comment (jonatns) [reply suministrospr/suministros/templates/suministros/suministro_detail.html]: Updated! @jpadilla

**Review by gcollazo — APPROVED** [review]
LGTM

**Review by jpadilla — APPROVED** [review]

**Comment by gcollazo** [comment]
Gracias @jonatns

**Pull request merged**